BlueElectrum.js

```diff
@@ -346,7 +347,7 @@ module.exports.estimateFee = async function(numberOfBlocks) {
 module.exports.getDustValue = async () => {
   const relayFee = await mainClient.blockchain_relayfee();
   // magic numbers from electrum vault
-  return (183 * 3 * relayFee) / 1000;
+  return btcToSatoshi((183 * 3 * relayFee) / 1000, 0);
```

Comment:
can we try not use "magic number" ?

Reply:
I'd love to but it is the response I got when I asked from what these numbers come from in the electrum vault so I thought I would at least write a comment

Reply:
ok :)